ci/scenarios/vPool/add_remove_alba_vpool_test/main.py

```diff
@@ -148,13 +150,16 @@ def validate_add_extend_remove_vpool(timeout=ADD_EXTEND_REMOVE_VPOOL_TIMEOUT):
 
         for cfg_name, cfg in vpool_configs.iteritems():
             # Create vpool
+            block_cache_cfg = None
+            if SystemHelper.get_ovs_version(SSHClient(SystemHelper.get_local_storagerouter(), username='root')).lower() == 'ee':
```

Comment:
get_ovs_version needs the sshclient param. But I don't need it further in the code so no need to store it in memory.